lib/pbkit/pbkit.c

```diff
@@ -2464,9 +2402,9 @@ void pb_push_transposed_matrix(DWORD *p, DWORD command, float *m)
     *((float *)p++)=m[_24];
     *((float *)p++)=m[_34];
     *((float *)p++)=m[_44];
-}
-
 
+    return p+17;
```

Comment:
This doesn't look right...We incremented the pointer while populating, now we are returning (p+16)+17

Reply:
Good eye, thanks!